### PR TITLE
[5897] Increase the maximum amount of user's and group's displayed in the Group/User Forms

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5897

### CODE REVIEW

File: ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx [hunk 1]
```diff
@@ -197,16 +204,21 @@ export function EditGroupDialogUI(props: GroupEditDialogUIProps) {
             members && (
               <>
                 <Typography variant="subtitle1" className={classes.sectionTitleEdit}>
-                  <FormattedMessage id="editGroupDialog.groupMembers" defaultMessage="Group Members" />
+                  <FormattedMessage id="editGroupDialog.editGroupMembers" defaultMessage="Edit Group Members" />
```
Comment (rart) [inline ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx]: Bad merge, "Group Members" is the latest

File: ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx [hunk 2]
```diff
@@ -197,16 +204,21 @@ export function EditGroupDialogUI(props: GroupEditDialogUIProps) {
...
                 </Typography>
                 <TransferList
                   disabled={group.externallyManaged}
+                  externallyManaged={true}
```
Comment (rart) [inline ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx:211]: Transfer list is not meant to be for groups should have no knowledge of group-specific concepts

File: ui/app/src/components/TransferList/TransferList.tsx [hunk 3]
```diff
@@ -14,22 +14,19 @@
...
-import NavigateBeforeIcon from '@mui/icons-material/NavigateBefore';
-import useStyles from './styles';
 import { createLookupTable } from '../../utils/object';
-import TransferListColumn, { TransferListItem } from '../TransferListColumn';
-import { FormattedMessage } from 'react-intl';
-import Tooltip from '@mui/material/Tooltip';
+import { TransferListItem } from '../TransferListColumn';
+import { LookupTable, PaginationOptions } from '../../models';
+import TransferListInternallyManaged from './TransferListInternallyManaged';
+import TransferListExternallyManaged from './TransferListExternallyManaged';
```
Comment (rart) [inline ui/app/src/components/TransferList/TransferList.tsx:22]: ?

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by jvega190** [comment]
Closing this PR in favor of https://github.com/craftercms/studio-ui/pull/3244

**Pull request closed without merging**